modules/code-editor/src/views/full/FileNavigator.tsx

```diff
@@ -77,14 +83,87 @@ export default class FileNavigator extends React.Component<Props, State> {
     }
   }
 
+  handleContextMenu = (node: ITreeNode<NodeData>, path, e) => {
+    if (!node.nodeData) {
+      return null
+    }
+
+    e.preventDefault()
```

Comment:
Just wondering if it would make sense to move the preventDefault to the top. If you right click a folder, you get the classical menu, but not if you click leaf node. If the tree handles context menu, I think it should just display nothing if a part of it doesn't use the feature